pom.xml

```diff
@@ -161,6 +161,9 @@
 
         <version.commons-digester>1.8.1</version.commons-digester>
         <maven.surefire.plugin.version>2.6</maven.surefire.plugin.version>
+
+        <version.org.springframework>3.2.12.RELEASE</version.org.springframework>
```

Comment:
This version property should already come from parent (jboss-integration-platform-parent)